guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<p>This is actually a bug. Can we move this to bug fixes (also not under arcgis.raster.utils). </p><ul><li>arcgis.raster</li><li class="ql-indent-1">ImageryLayer</li></ul><p>					  Fix the compute_histograms() function to properly work with time extent passed as a list of datetime.date objects</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Done! Thank you!